cypress/integration/onboarding/messaging/message_pinning_unpinning_spec.js

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [number] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+/*eslint max-nested-callbacks: ["error", 5]*/
+
+const pinnedPosts = [];
+
+/**
+* Pin post by cliking on [...] then 'Pin to channel', and add the pinned post to pinnedPosts
+* @param {String} postId - post ID of the post to pin
+*/
+function pinPost(nthPost) {
+    cy.getNthPostId(nthPost).then((postId) => {
+        cy.clickPostDotMenu(postId);
+        cy.get(`#pin_post_${postId}`).click();
+        pinnedPosts.push(postId);
+    });
+}
+
+describe('Messaging', () => {
+    before(() => {
+        // Login and go to /
+        cy.apiLogin('user-1');
+        cy.visit('/');
+    });
+
+    // Unpin all posts at the end of the test
+    after(() => {
+        pinnedPosts.forEach((pinnedPost) => {
+            cy.apiUnpinPosts(pinnedPost);
+        });
+    });
+
+    it('M15010 Pinning or un-pinning older post does not cause it to display at bottom of channel', () => {
+        cy.get('#sidebarItem_saepe-5').click();
```

Comment:
For some reason, this is failing on me.  One way is to do force click [`({force: true})`]

Reply:
Added the forced click there. Hopefully it'll run now :)